cor-journey-test-data/src/main/resources/tps/testdata/cotax/journey-3-EnteredPayment.json

```diff
@@ -12,20 +12,6 @@
     "updated" : "2059-11-25T16:33:51.880Z",
     "customerName" : "customerName",
     "chargeReference" : "1097172564A00104A",
-    "pcipalData" : {
```

Comment:
Mostly the changes are removing this data from journey datas that are before we receive notification from pcipal, since we don't have it until that point